packages/transaction-controller/src/types.ts

```diff
@@ -68,6 +68,11 @@ type TransactionMetaBase = {
    */
   blockTimestamp?: string;
 
+  /**
+   *
+   */
+  bridgeSteps?: BridgeStep[];
```

Comment:
To clarify, do we definitely need to store this here? We don't have a bridge specific controller that will persist this state also and can link to a transaction ID?